src/main/java/org/teamresistance/frc/IO.java

```diff
@@ -17,6 +18,9 @@
 
   public static final Gyro3D gyro = new Gyro3D(SPI.Port.kMXP);
 
+  public static final Solenoid grabSolenoid = Hardware.Solenoids.doubleSolenoid(1, 2, Solenoid.Direction.EXTENDING);
+  public static final Solenoid armSolenoid = Hardware.Solenoids.doubleSolenoid(3, 4, Solenoid.Direction.EXTENDING);
```

Comment:
On second thought, the Strongback `Solenoid` interface isn't right for what we're doing―it only makes sense when applied to DoubleSolenoids, like what had Shreya said. She and I had a brief private discussion and decided ideally we would be using an interface like this one:

```java
public interface Solenoid {
  void extend();
  void retract();
  boolean isExtended();
  boolean isRetracted();
}
```

If you're feeling adventurous, @BreeCat10, do you think you could try your hands at writing a class that has those methods that forwards the calls to an internal/private WPILib `Solenoid` instance? You would be writing a wrapper.